core-tests/src/test/java/org/mule/runtime/core/processor/strategy/AbstractProcessingStrategyTestCase.java

```diff
@@ -527,4 +577,22 @@ public Event process(Event event) throws MuleException {
   public static Matcher<Long> between(long min, long max) {
     return allOf(greaterThanOrEqualTo(min), lessThanOrEqualTo(max));
   }
+
+  protected TypeSafeMatcher<MessagingException> overloadErrorTypeMatcher() {
```

Comment:
There is an `ExpectedError` `Rule` already but it doesn't play nicely with `ExpectedException` :-(